includes/class-sensei-course.php

```diff
@@ -524,10 +525,11 @@ private function save_post_meta( $post_key = '', $post_id = 0 ) {
 		 * Action before saving the meta value.
 		 *
 		 * @since 2.2.0
+		 * @hook sensei_course_meta_before_save
 		 *
-		 * @param int    $post_id        The course ID.
-		 * @param string $meta_key       The meta to be saved.
-		 * @param mixed  $new_meta_value The meta value to be saved.
+		 * @param {int}    $post_id        Course ID.
+		 * @param {string} $meta_key       The meta to be saved.
+		 * @param {mixed}  $new_meta_value The meta value to be saved.
```

Comment:
Have you checked how `{mixed}` renders in the docs? I've seen some places that `{*}` is used in JSDoc for that, but I'm not sure.

Reply:
It looks OK:
![Screen Shot 2020-08-31 at 6 38 23 AM](https://user-images.githubusercontent.com/1190420/91711846-a074a280-eb54-11ea-80bb-eb243b4f813b.jpg)